site/content/reference/01-commands.md

```diff
@@ -0,0 +1,235 @@
+---
+title: "Commands Reference"
+weight: 10
+---
+
+## Commands Reference
+
+```
+a CLI for Amazon EKS
+
+Usage: eksctl [command] [flags]
+
+Commands:
+  eksctl completion  Generates shell completion scripts
+  eksctl create      Create resource(s)
+  eksctl delete      Delete resource(s)
+  eksctl drain       drain resources(s)
+  eksctl get         Get resource(s)
+  eksctl help        Help about any command
+  eksctl scale       Scale resources(s)
+  eksctl update      Update resource(s)
+  eksctl utils       Various utils
+  eksctl version     Output the version of eksctl
+
+Use 'eksctl [command] --help' for more information about a command.
+```
+
+`Usage: eksctl [command] [flags]`
+
+##### common flags<a name="common-flags"></a>
+
+These flags can be applied to all eksctl commands and behave consistently between them.
+
+- [C, color](02-flags.md#color)
+- [h, help](02-flags.md#help)
+- [v, verbose](02-flags.md#verbose)
+
+### completion <a name="completion"></a>
```

Comment:
This is very linux centric. Do we really need to document this behaviour here?

Reply:
I think Linux users would appreciate some documentation, yes. 😄 

It's also supposed to cover MacOS usage, but I think it might be inaccurate.  😐 

Reference documentation is supposed to be comprehensive, so if the code supports a feature you'll want to document that feature - no matter how trivial it may seem.  If it supports tab completion for a Unix/Linux shell, it should really be documented.

This is literally cut-and-paste from the help screens, so it was easy to include.  I don't know why you wouldn't.